.github/containerscan/allowedlist.yaml

```diff
@@ -1,10 +1,11 @@
 ---
 general:
   vulnerabilities:
-  - CVE-2022-25857  # snake yaml, fixed in 1.31
-  - CVE-2022-45868 #h2
-  - CVE-2016-1000027 # spring-web deserialization. From year 2016, fixed in 6.0.0
-  - CVE-2022-1471 # snake yaml, fixed in 2.0 (new spring boot version?) https://bitbucket.org/snakeyaml/snakeyaml/wiki/CVE-2022-1471
```

Comment:
kjekt å få med kommentarane... for folk?

Reply:
enig... trykte berre knappen i github eg... 😞

Reply:
rekna med det, gjort akkurat det same sjøl! :)